tools/shared.py

```diff
@@ -485,14 +486,10 @@ def sanity_is_correct():
       pass
     if sanity_data == expected:
       logger.debug(f'sanity file up-to-date: {sanity_file}')
-      # Even if the sanity file is up-to-date we still need to at least
-      # check the llvm version. This comes at no extra performance cost
-      # since the version was already extracted and cached by the
-      # generate_sanity() call above.
+      # Even if the sanity file is up-to-date we still run the checks
+      # when force is set.
       if force:
         perform_sanity_checks()
-      else:
-        check_llvm_version()
```

Comment:
Can we not check the LLVM version when doing sanity checks during link? That's what I thought you meant when I read the PR description.

Reply:
Yes, that was my intent.